samples/azure-quantum/hidden-shift/README.md

```diff
@@ -46,3 +46,4 @@ az quantum target list --output table
 
 - [HiddenShift.csproj](https://github.com/microsoft/quantum/blob/main/samples/azure-quantum/hidden-shift/HiddenShift.csproj): Main Q# project file for this sample.
 - [HiddenShift.qs](https://github.com/microsoft/quantum/blob/main/samples/azure-quantum/hidden-shift/HiddenShift.qs): Main Q# program for this sample.
+- [hidden-shift.ipynb](./hidden-shift.ipynb): Azure Quantum notebook for running the sample
```

Comment:
It looks like this notebook uses Qiskit + Python instead of Q# + Python — that should be made clear in the description, and in the manifest. The sample notebook itself should ideally point to the Q# version to avoid conveying the impression that Qiskit + Python is preferred over Q# + Python workflows. That is especially critical here since the Q# program is standalone, but the Qiskit one uses Python — setting those up as alternatives could lead users to erroneously believe that Q# and Python are to be considered as orthogonal.

Reply:
Would you be able to update this README with the preferred wording? Or we can ping Mathias when he gets back.

For the sample notebook itself, I don't think we should point to the Q# local sample, since it directs user away from cloud experience, where this notebook is hosted.

Reply:
> Would you be able to update this README with the preferred wording? Or we can ping Mathias when he gets back.

As an initial stab:

```suggestion
- [hidden-shift.ipynb](./hidden-shift.ipynb): Python + Qiskit version demonstrating usage with the Azure Quantum service.
```

We will still need to modify the notebook itself as well, though. As it stands, this sets up an alternative between Q# standalone and Qiskit + Python, which reinforces a negative confusion that we have seen from our users — pointing to Q# + Python as an option is a hard requirement here, and not something we can work around.

> For the sample notebook itself, I don't think we should point to the Q# local sample, since it directs user away from cloud experience, where this notebook is hosted.

The sample is not just hosted in the portal gallery, to be fair, but also in all other contexts that samples appear in (to wit: GitHub, samples gallery, zero-install). I understand not directing users away from the portal, but that cannot come at the cost of creating user confusion by implying that Q# and Python are orthogonal.

As a prerequisite for merging, I think we need to find a way to point to other sample content, even if only in prose — that's a frustrating UX, but frustrating is less bad than actively harmful.

Reply:
Ok, added a bullet to the end of the notebook linking back to Q# standalone as well as telling user they can create a python host program.